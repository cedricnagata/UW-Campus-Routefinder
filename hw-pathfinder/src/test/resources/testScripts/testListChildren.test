# Test listing children of node

CreateGraph Graph

AddNode Graph A
AddNode Graph B
AddNode Graph C
AddNode Graph D

AddEdge Graph A B 1.0
AddEdge Graph A C 1.0
AddEdge Graph A D 1.0

ListChildren Graph A