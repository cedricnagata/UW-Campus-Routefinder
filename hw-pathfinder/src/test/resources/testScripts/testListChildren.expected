# Test listing children of node

created graph Graph

added node A to Graph
added node B to Graph
added node C to Graph
added node D to Graph

added edge 1.000 from A to B in Graph
added edge 1.000 from A to C in Graph
added edge 1.000 from A to D in Graph

the children of A in Graph are: B(1.0) C(1.0) D(1.0)